tool/site/client-react/src/pages/myPrescriptionsPage.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from "react"
+import {useContext} from "react"
+import {Label, Table} from "nhsuk-react-components"
+import {AppContext} from "../index"
+import LongRunningTask from "../components/longRunningTask"
+import {axiosInstance} from "../requests/axiosInstance"
+import PrescriptionActions from "../components/prescriptionActions"
+
+const MyPrescriptionsPage: React.FC = () => {
+  const {baseUrl} = useContext(AppContext)
+  const retrievePrescriptionsTask = () => retrievePrescriptions(baseUrl)
+
+  return (
+    <LongRunningTask<Prescriptions> task={retrievePrescriptionsTask} loadingMessage="Retrieving prescriptions.">
+      {prescriptions => {
+        if (!prescriptions.any) {
+          return (
+            <>
+              <Label isPageHeading>My Prescriptions</Label>
+              <p>You do not have any active prescriptions. Once you create or release a prescription you will see it here</p>
+            </>
+          )
+        }
+        return (
+          <>
+            <Label isPageHeading>My Prescriptions</Label>
+            <PrescriptionGroupTable
```

Comment:
thoughts on just having the one table that renders all prescriptions and their current status and decides the actions that a prescription can have based on its status?

Reply:
did try that originally, found it harder to use, as when you're performing prescription actions you want to easily see current states of prescriptions grouped together. I'd suggest use it in anger for a while before deciding what you want to change about it